woltka/cli.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python3
+
+# ----------------------------------------------------------------------------
+# Copyright (c) 2020--, Qiyun Zhu.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+"""Standalone command-line interface (CLI) of the program.
+"""
+
+import click
+
+from . import __version__
+from .workflow import workflow
+
+
+class NatOrder(click.Group):
+    """Natural ordering of click command groups.
+    """
+    def list_commands(self, ctx):
+        return self.commands.keys()
+
+
+@click.version_option(__version__)
+@click.group(cls=NatOrder)
```

Comment:
If you accept the change above.

```suggestion
@click.group(cls=NaturalOrderGroup)
```